File: Content/Mods/StyxScribeSharedREPL/StyxScribeSharedREPL.lua
ModUtil.Mod.Register( "StyxScribeSharedREPL" )

function StyxScribeSharedREPL.Define( )
	StyxScribeShared.Root.RunLua = StyxScribeShared.Async( StyxScribeREPL.RunLua )
end

StyxScribe.AddHook( StyxScribeSharedREPL.Define, "StyxScribeShared: Reset", StyxScribeSharedREPL )

StyxScribeSharedREPL.Define( )

File: Content/Mods/StyxScribeSharedREPL/StyxScribeSharedREPL.py
def Define(message=None):
    REPL = Scribe.Modules.StyxScribeREPL
    Shared = Scribe.Modules.StyxScribeShared
    Shared.Root.RunPython = Shared.Async(REPL.RunPython)

Priority = 0

def Load():
    Scribe.AddHook(Define, "StyxScribeShared: Reset", __name__)
